src/service/video-manager-impl.js

```diff
@@ -367,6 +428,13 @@ class VideoEntry {
     }
     if (this.hasDocking) {
       this.dockableVideoBuilt_();
+      // Determine the page's direction to help decide which side to dock on
+      // TODO(@wassgha) Probably will be needed for more functionality later
+      // but for now, only needed for video docking
+      const doc = this.ampdoc_.win.document;
+      this.pageDir_ = doc.body.getAttribute('dir')
```

Comment:
nit: we are doing this in a few place. (amp-sidebar,amp-selector), let's do a `isRTL(win)` in `dom.js`?